.github/workflows/go.yml

```diff
@@ -34,6 +34,34 @@ jobs:
     - name: Test
       run: go test -v ./...
 
+  cover:
+    name: Coverage
+    runs-on: ubuntu-22.04
+
+    env:
+      CGO_ENABLED: 0
+      GOEXPERIMENT: nocoverageredesign
```

Comment:
Likely can be avoided here.

Reply:
Can't.

Reply:
Well, it can't on my machine. But this action uses Go 1.23, so may be we should try.